PowerPointLabs/PowerPointLabs
# #1691 Format Tab gets focus even when maintain tab focus tab is ticked

Fixes #1691 

Limitations:
- the ribbon only load once, even if new presentations are open, hence there is only a single "Format tab" object. As a result of that, the behaviour faced in the issue because of different presentations setting the visibility of the same Format Tab differently.

Solution
- Since the ribbon load only once, to set the boolean value when ribbon loads, and sync it across all the open/new presentations whenever there a checkbox in one of the presentation is clicked. Meaning if there is Presentation 1/2/3, and the checkbox is ticked for presentation 2, when user goes to presentation 1/3, the checkbox would reflect the same ticked status.

Tested on multiple new/open presentations in 2016 and it works.

To Test
- Open multiple new/existing presentations and see if they sync when you check the box on one of them
